### PR TITLE
Events and event pages

### PR DESCRIPTION
**Pull request by MikaelSiidorow**
- [x] events listing page
- [x] event specific page
  - [x] open quota and waitlist 
- [x] i18n
- [ ] all corner cases
- [ ] error handling

Progresses towards #224

### CODE REVIEW

File: apps/web/src/lib/api/external/ilmomasiina/index.ts
Comment (PurkkaKoodari) [inline apps/web/src/lib/api/external/ilmomasiina/index.ts:1]: Depend on `@tietokilta/ilmomasiina-models` to get the types directly from the source. (this is also pushed to NPM by the ilmomasiina repo workflows)

File: apps/web/src/lib/utils.ts [hunk 1]
```diff
@@ -43,3 +50,129 @@ export const insertSoftHyphens = (text: string): string => {
...
+  const formatter = new Intl.DateTimeFormat(`${locale ?? "fi"}-FI`, {
+    day: "numeric",
+    month: "numeric",
+    year: "numeric",
+  });
+
+  return formatter.format(new Date(date));
+};
+
+export const getQuotasWithOpenAndQueue = (
```
Comment (PurkkaKoodari) [inline apps/web/src/lib/utils.ts:117]: I think this stuff is also available in `@tietokilta/ilmomasiina-components` - might be split in the future to provide such utility code without any UI deps

File: apps/web/src/lib/api/external/ilmomasiina/index.ts
Comment (MikaelSiidorow) [reply apps/web/src/lib/api/external/ilmomasiina/index.ts:1]: I took a look but the typebox types and combining to a full request might take more effort, someone can take a look a this later but current implementation works fine for now

File: apps/web/src/lib/utils.ts (hunk 1 above)
Comment (MikaelSiidorow) [reply apps/web/src/lib/utils.ts:117]: Yeah, there was something but not exactly the same. Might make sense to unify later but not a big deal for such small functionality

**Pull request merged**